README.md

```diff
@@ -96,41 +151,59 @@ Click on image to view fullscreen and zoom
 
 ## Setup and Running
 - Prerequisites
-  - Node
-  - MySQL (or Postgres / Sqlite / MSSQL)
-- Clone repo `git clone git@github.com:atulmy/crate.git crate`
+  - Node -- information can be found [here](https://treehouse.github.io/installation-guides/mac/node-mac.html)
+  - MySQL (or Postgres / Sqlite / MSSQL) -- information can be found [here](https://postgresapp.com/) (Note: user may need to force quit and restart terminal and/or computer for success)
+  - Once you have postgreSQL installed, complete the following:
+  - Run `psql` in your terminal to access the postrgres command line interface (CLI)
+  - Check your available postgres users by running `\du` in the postgres CLI. Your available users are listed under “Role name.” Use this user as your username in the `database.json file`
+  - In `psql`, create your database with the command `CREATE DATABASE crate;` 
+
+
+- Clone repo `git clone git@github.com:KathyBui732/crate.git crate`
 - Switch to `code` directory `cd code`
 - Configurations
-  - Modify `/api/src/config/database.json` for database credentials
+  - Modify `/api/src/config/database.json` for database credentials -- assign either `postgres` or your `<username>` values to `username` keys, aforementioned above. Assign `postgres` value to `dialect` key. 
```

Comment:
I think that instead of actually listing the errors we run into on the readme, we should just have like an errors? link or something that they can click on next to the step that links to the issues. I opened all the issues we ran into on our project board. That way the instructions are clearer and people can go to the issue only if they need it.

Reply:
Sounds great!!

Reply:
I like that idea too!